guides/source/upgrading_ruby_on_rails.md

```diff
@@ -535,7 +535,7 @@ ActiveSupport::JSON::Encoding.time_precision = 0
 Previously, Rails allowed inline callback blocks to use `return` this way:
 
 ```ruby
-class ReadOnlyModel < ActiveRecord::Base
+class ReadOnlyModel < ApplicationRecord
```

Comment:
@dhh - Please reopen this PR. I have updated it with the changes only in this file.